distribution/test/itests/README.md

```diff
@@ -16,6 +16,16 @@ In short:
 * Use remote debugging to debug tests themselves or anything related to production code
 * Use local debugging to debug the code that sets up and configures the testing environment
 
+## Solr
+Solr runs in its own OS process separate PAX-EXAM and the application. Things to know include:
+
+* The default port for Solr is hardcoded to `9994`. 
+* The Solr administration UI is available at `http://localhost:9994/solr`
```

Comment:
❓ Can you actually hit this during the itest run?

Reply:
Sure.